packages/loot-core/src/server/accounts/sync.ts

```diff
@@ -77,27 +77,27 @@ export async function getAccounts(userId, userKey, id) {
 
 export async function getNordigenAccounts(userId, userKey, id) {
   const userToken = await asyncStorage.getItem('user-token');
-  if (userToken) {
-    let res = await post(
-      getServer().NORDIGEN_SERVER + '/accounts',
-      {
-        userId,
-        key: userKey,
-        item_id: id,
-      },
-      {
-        'X-ACTUAL-TOKEN': userToken,
-      },
-    );
+  if (!userToken || !getServer()) return;
```

Comment:
Sure! Do you think it would make sense to have it be a function like `await getUserTokenForServer()` that would return `null` if `getServer()` returns no server?

Reply:
I might be reading this wrong, but isn't the operation we're performing here just an assertion that the server is set up and the user has authenticated?

if that's so, then perhaps: `async isLoggedIn(): boolean`?

Reply:
Yes, but we usually need to pass the `userToken` to the server. The conditional would be shorter but then TS wouldn't be able to narrow the type of `userToken` to be non-null.

Reply:
Ok, then your suggestion makes more sense.

But now that i'm thinking about this more.. why do we have `userToken` set if the server is not set? Could we instead remove it if server is not set-up? Thus making the scope of this PR much smaller.

Reply:
Updated!